### PR TITLE
Cron entry to keep our logs from accumulating.

### LINKED ISSUE #12736: Integrate `logrotated` into Airflow V2 stack
Airflow stores the output of DAG runs and other stateful information in the form of many, many text files in a rich hierarchy of directories. Presently, on our data03 stack in the `logs` folder, the following is reported:

```
# find | wc
 125956  125956 12917878
# du -sch *
7.2M	dag_id=atd_service_bot_issue_intake_production
20K	dag_id=test_slack_notifier_production
386M	dag_id=weather-checker
73M	dag_processor_manager
111M	scheduler
576M	total
```

This indicates 125K or so files & directories totaling a half gig of data. This is a non-trivial amount, and we can expect this size to grow rapidly. 

This issue is to track the configuration of `logrotated` to daily archive these and ultimately remove the archives after a configured amount of storage time. 

- [ ] add documentation to readme

CC @mddilley & @johnclary

### PR DESCRIPTION
**Pull request by frankhereford**
## Associated issues

This PR intends to close https://github.com/cityofaustin/atd-data-tech/issues/12736.

## Testing

Jump on `atd-data03` and as your own, non-root, user, do the following (but keep an eye on `/home` capacity and the system load because `atd-data03` runs hot):

```bash
mkdir ~/your_test_directory;

rsync -av \ # this gets a representative sample of what logs look like
/usr/airflow/atd-airflow/logs/dag_id=*exec* \
/usr/airflow/atd-airflow/logs/*processor* \
/usr/airflow/atd-airflow/logs/scheduler  \
~/your_test_directory/logs

find ~/your_test_directory/logs -type f -mtime +30 -exec rm -v {} \;
find ~/your_test_directory/logs -type d -empty | xargs -I {} rmdir -v {};
find ~/your_test_directory/logs -type d -empty | xargs -I {} rmdir -v {}; 
```

Note that the empty directory clean-up command runs twice because there are directories holding directories that /become/ empty when the first one runs.

If you're satisfied with how it works, check the file in this PR and make sure you like how this is implemented.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Add note to 1PW secrets moved to API vault and check for duplicates

### CODE REVIEW

File: toolbox/log-purge/log-purge.cron [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Cron job to clean up old files and empty directories from /usr/airflow/atd-airflow/logs
+# This will run nightly at 2 AM. The first command removes all files which have a modification
+# time greater than 30 days, and the second command removes all empty directories. Removed
+# files and directories are logged into /var/log which will get rotated by the system.
+
+0 2 * * * find /usr/airflow/atd-airflow/logs -type f -mtime +30 -exec rm -v {} \; \
```
Comment (frankhereford) [inline toolbox/log-purge/log-purge.cron:6]: 2 AM sound like a good time to do the rotation?

Comment (frankhereford) [inline toolbox/log-purge/log-purge.cron:6]: And also, how long back should we keep stuff? I went with 30 days here.

Comment (johnclary) [reply toolbox/log-purge/log-purge.cron:6]: sounds great!

Comment (johnclary) [reply toolbox/log-purge/log-purge.cron:6]: sounds great. i can't imagine needing a longer history than that! plus the Airflow DB itself keeps a longer history, right?

Comment (frankhereford) [reply toolbox/log-purge/log-purge.cron:6]: I'm not sure about the DB history TBH, but my next issue is to start backing up the DB, so I'll get a handle on what data we actually have in there when I do that and report back.

**Review by johnclary — APPROVED** [review]
thank you @frankhereford for knocking this out!

**Review by Charlie-Henry — APPROVED** [review]
Nice! Ty @frankhereford

**Review by chiaberry — APPROVED** [review]

**Pull request merged**